src/test/html/html-script-scanner_test.ts

```diff
@@ -63,5 +69,37 @@ suite('HtmlScriptScanner', () => {
       assert.equal(feature0.type, 'html-script');
       assert.equal(feature0.url, '/aybabtu/foo.js');
     });
+
+    suite('modules', () => {
+      const urlLoader = new FSUrlLoader(fixturesDir);
+      const analyzer = new Analyzer({urlLoader});
+      let analysis: Analysis;
+
+      before(async() => {
+        analysis = await analyzer.analyze(['js-modules.html']);
+      });
+
+      test('finds external module scripts', () => {
+        const htmlScripts = [...analysis.getFeatures({kind: 'html-script'})];
+        assert.equal(htmlScripts.length, 1);
+        const js = <JavaScriptDocument>htmlScripts[0].document.parsedDocument;
```

Comment:
style nit: prefer `as` to cast

Reply:
Done.  I also went ahead and verified the contents of the document contents in the tests to ensure each referred to the expected material.  I also added the missing `./submodule.js` file to verify full import graph traversal.